utils/build/build.js

```diff
@@ -250,14 +250,14 @@ for (const pkg of workspace.packages()) {
   steps.push({
     command: 'npx',
     args: [
-      'babel',
-      ...(watchMode ? ['-w'] : []),
-      ...(withSourceMaps ? ['--source-maps'] : []),
-      '--extensions', '.ts',
-      '--out-dir', quotePath(path.join(pkg.path, 'lib')),
-      '--ignore', '"packages/playwright-core/src/server/injected/**/*"',
```

Comment:
Where did this go?

Reply:
This does indeed build the injected scripts twice. Once with esbuild in Node.js mode and once with esbuild with browser target. Looks like that esbuild has no support for `--ignore` cli flag - maybe we can solve it differently.

Reply:
Injected scripts were moved into its own package in https://github.com/microsoft/playwright/pull/35483, so this option is not needed any more.